src/actions/sources/setSourceLocation.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import type { Action } from "../types";
+
+export const setSourceLocation: Action = (url, options) => ({
+  type: "SET_SOURCE_LOCATION",
+  url: url,
+  line: options.location ? options.location.line : null
+});
```

Comment:
Maybe we can move this to a common file with other functions? so we don't end up with many single files with only one function in them.

Reply:
Sounds like a great idea, there are a few of these actions and having 20 files sounds like a bit too much - what would you call that file?